CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Master
 
+#### Fixed
+- Quote `FRAMEWORK_SEARCH_PATHS` for `framework` dependencies [382](https://github.com/yonaskolb/XcodeGen/pull/382) @brentleyjones
```

Comment:
Maybe this could be the problem instead of the solution: 
Fix `FRAMEWORK_SEARCH_PATHS` for `framework` dependency paths with spaces